Makefile

```diff
@@ -22,7 +22,7 @@ ifeq ($(ARCH),)
 endif
 CURRENT_OS=$(shell go env GOOS)
 CURRENT_OS_ARCH=$(shell echo $(CURRENT_OS)-`go env GOARCH`)
-GOBIN?=$(shell echo `go env GOPATH`/bin)
+GOBIN?=$(shell echo `go env GOPATH|cut -d: -f1`/bin)
```

Comment:
"It's probably a simple one-line change"

^ Me, three weeks ago. (Yes it was! But how does one know which line needs to change? The error was: https://app.circleci.com/pipelines/github/fluxcd/flux/976/workflows/dd384c61-c331-4a9b-9154-d3ea5dd872af/jobs/11947?invite=true#step-107-132
```
Makefile:40: *** target pattern contains no '%'.  Stop.
```

FFS... I think we might want to consider again modernizing the CircleCI / moving off and adopting GitHub Actions for Flux v1, as originally proposed in #2944, now that I am more familiar with our build tooling and with GitHub Actions, I could consider reopening that issue here.)

Not today, this one-line kludge means we can keep using it for now.